docs/src/main/sphinx/release/release-435.md

```diff
@@ -0,0 +1,75 @@
+# Release 435 (13 Dec 2023)
+
+## General
+
+* Add support for the `json_table` table function. ({issue}`18017`)
+* Reduce coordinator memory usage. ({issue}`20018`, {issue}`20022`)
+* Increase reliability and memory consumption of inserts. ({issue}`20040`)
+* Fix incorrect results for `LIKE` with some strings containing repeated
+  substrings. ({issue}`20089`)
+* Fix coordinator memory leak. ({issue}`20023`)
+* Fix possible query failure for `MERGE` queries when `retry-policy` set to
+  `TASK` and `query.determine-partition-count-for-write-enabled` set to `true`.
+  ({issue}`19979`)
+* Prevent hanging query processing with `retry.policy` set to `TASK` when a
+  worker node died. ({issue}`18603 `)
+* Fix query failure when reading array columns. ({issue}`20065`)
+
+## Delta Lake connector
+
+* {{breaking}} Remove support for registering external tables with
+  `CREATE TABLE` and the `location` table property. Use the
+  `register_table` procedure as replacement. The property
+  `delta.legacy-create-table-with-existing-location.enabled` is
+  also removed. ({issue}`17016`)
+* Improve query planning performance on Delta Lake tables. ({issue}`19795`)
+* Ensure AWS access keys are used for connections to the AWS Security Token
+  Service. ({issue}`19982`)
+* Reduce memory usage for inserts into partitioned tables. ({issue}`19649`)
+
+## Elasticsearch connector
+
+* Ensure certificate validation is skipped when
+  `elasticsearch.tls.verify-hostnames` is `false`. ({issue}`20076`)
+
+## Hive connector
+
+* Support coercion from numeric integer to decimal types with Hive schema
+  evolution. ({issue}`19931`)
+* Support coercion from date to varchar types with Hive schema evolution. ({issue}`19500`)
```

Comment:
Check previous release notes:

"Add support for columns that changed from `timestamp` to `date` type"

"Add support for reading tables where a column’s type has been changed from `boolean` to `varchar`"

The fact that it's doing a coercion is an implementation detail that doesn't need to be discussed.

Reply:
ok